atomic_reactor/plugins/post_export_operator_manifests.py

```diff
@@ -125,11 +128,31 @@ def run(self):
                     filepath = os.path.join(filedir, f)
                     archive.write(os.path.join(root, f), filepath, zipfile.ZIP_DEFLATED)
             manifest_files = archive.namelist()
-            if not manifest_files:
```

Comment:
Why was this removed?

Reply:
Added it back. It was removed due to the `_verify_csv` was called without checking the operator bundle manifests. Thanks for pointing out this.